target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -242,6 +242,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "0:wan" "6@eth0"
 		;;
+	kng_rc)
+		ucidef_add_switch "switch1" \
```

Comment:
No. There are two of them in device. Switch0 - unmanaged mt7530 and managed switch1 rtl8367rb.